### PR TITLE
Add ActiveBrokers() method to Kafka AsyncProducer

### LINKED ISSUE #169: Kafka: add support for active brokers in Producers
## Is your feature request related to a problem? Please describe

It is necessary to check the healthiness of Kafka consumers/producers; in https://github.com/Shopify/sarama/issues/1341 it is mentioned that this could be done by checking that there are active brokers (they are checked periodically by the client anyways).

## Describe the solution

The minimally invasive solution is to use https://godoc.org/github.com/Shopify/sarama#NewAsyncProducerFromClient in Patron and then expose the `Brokers()` method.

~This solution should work for both consumers and producers if we expose functionality of the `Client`.~
We will focus on the producers only.

## Additional context

This is related to #148, in the sense that more functionality of the Kafka client library is necessary.

### PR DESCRIPTION
**Pull request by gm42**
This PR adds an `ActiveBrokers()` method to Kafka `AsyncProducer`. It also expands the GoDoc comment for Close() method and fixes a typo in an error message.

## Which problem is this PR solving?

Fixes #169

### CODE REVIEW

**Review by mantzas — DISMISSED** [review]

**Comment by mantzas** [comment]
@gm42 you have breaking changes due to a refactoring of packages. All client code has now been moved to the client package.

**Review by drakos74 — DISMISSED** [review]

File: trace/kafka/kafka.go [hunk 1]
```diff
@@ -98,11 +99,27 @@ func (ap *AsyncProducer) Error() <-chan error {
 	return ap.chErr
 }
 
-// Close gracefully the producer.
+// ActiveBrokers returns a list of active brokers' addresses.
+func (ap *AsyncProducer) ActiveBrokers() []string {
+	brokers := ap.prodClient.Brokers()
+	activeBrokerAddresses := make([]string, len(brokers))
+	for i, b := range brokers {
```
Comment (drakos74) [inline trace/kafka/kafka.go:109]: Would you consider using „copy“ for a slice ? And let go do the memory re-arrangement under the hood ?

Comment (gm42) [reply trace/kafka/kafka.go:109]: @drakos74 can you make an example for this specific loop? I don't think `copy` can be used here.

**Comment by gm42** [comment]
Closed in favor of #192

**Pull request closed without merging**